dive_core/command_hierarchy.cpp

```diff
@@ -1419,7 +1379,8 @@ uint64_t CommandHierarchyCreator::AddRegisterNode(uint32_t       reg,
     for (uint32_t field = 0; field < reg_info_ptr->m_fields.size(); ++field)
     {
         const RegField &reg_field = reg_info_ptr->m_fields[field];
-        uint64_t        field_value = (reg_value & reg_field.m_mask) >> reg_field.m_shift;
+        uint64_t        field_value = (reg_value & reg_field.m_mask) >> reg_field.m_shift
```

Comment:
@shanminchao can you confirm this is correct? i checked the cffdump code, and they use the shr to shift left